Cubical/HITs/Hopf.agda

```diff
@@ -18,16 +38,286 @@ HopfS² (surf i j) = Glue S¹ (λ { (i = i0) → _ , idEquiv S¹
                                ; (j = i1) → _ , _ , rotIsEquiv (loop i) } )
 
 -- Hopf fibration using more direct definition of the rot equivalence
+-- TODO : prove that it is equivalent to HopfSuspS¹
 HopfS²' : S² → Set
 HopfS²' base = S¹
 HopfS²' (surf i j) = Glue S¹ (λ { (i = i0) → _ , rotLoopEquiv i0
                                 ; (i = i1) → _ , rotLoopEquiv i0
                                 ; (j = i0) → _ , rotLoopEquiv i0
                                 ; (j = i1) → _ , rotLoopEquiv i } )
 
--- Hopf fibration using suspension of S¹
-HopfSuspS¹ : SuspS¹ → Set
-HopfSuspS¹ north = S¹
-HopfSuspS¹ south = S¹
-HopfSuspS¹ (merid x i) = ua (_ , rotIsEquiv x) i
+-- Total space of the fibration
+TotalSpace : Set
+TotalSpace = Σ SuspS¹ HopfSuspS¹
+
+-- Forward direction
+filler-1 : I → (j : I) → (y : S¹) → Glue S¹ (Border y j) → join S¹ S¹
+filler-1 i j y x = hfill (λ t → λ { (j = i0) → inl (rotInv-1 x y t)
+                                  ; (j = i1) → inr x })
+                         (inc (push ((unglue (j ∨ ~ j) x) * inv y) (unglue (j ∨ ~ j) x) j)) i
```

Comment:
Indentation

Reply:
The bad indentation is caused by Github's weird font that contracts fi into only one character. I believe I should stick with more standard indentation ?

Reply:
I see, bad Github! So no need to change this then

Reply:
This indentation looks OK to me. The last line is the second argument of `hfill`, it's indented so that the first letter is at the same position of the first argument.